Atlas/frontend/src/components/AtlasLogin.js

```diff
@@ -0,0 +1,150 @@
+import React, { Component } from 'react';
+import {Button, Form, FormGroup, Input, Col, Row, Container, Badge} from 'reactstrap';
+import '../css/AtlasLogin.css';
+import atlas from '../assets/images/atlas.jpeg';
+import {AtlasSignup} from "../Components";
+import logo from '../assets/images/logo.png';
+import LoginSearch from "./LoginSearch";
+
+export default class AtlasLogin extends Component {
+
+    constructor(props){
+        super(props);
+        this.state = {username: '', password: ''};
+
+        this.handleNameChange = this.handleNameChange.bind(this);
+        this.handlePasswordChange = this.handlePasswordChange.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
+
+
+    handleSubmit(){
+        //request server and redirect to main page or fail to log in
+    }
+
+    handleNameChange(event){
+        this.setState({username: event.target.username});
+    }
+
+    handlePasswordChange(event){
+        this.setState({password: event.target.password});
+    }
+
+    handleResponse = (data) => {
+        console.log(data);
+    }
+
+    handleError = (error) => {
+        this.setState({ error });
+    }
+
+
+    render() {
+        return (
+        <Container>
+            <img src={atlas} className="background-image" alt="background" />
+            <LoginSearch logo={logo} />
+            <Row><h1></h1></Row>
```

Comment:
This can't be the way to do it :) You should utilise css margins instead of this.